### ISSUE TITLE
no internet detected on selected network interface(nethunter)

### ISSUE THREAD

**Issue by lightakun** [v1s1t0r1sh3r3/airgeddon#109]
I am using kali nethunter in my leeco le2 x526 with the latest version of airgeddon 7.2.3.
Everything seems to work fine(monitor mode and all) but whenever i try to do evil twin with internet access(with selected my network interface with internet access as rm_datanet0), it doesnt detects internet on it :( . Heres the ping on the selected interface.(ping -I rmdata_net0 github.com)

https://pastebin.com/L6iVk1xN

This interface has internet access...

Waiting fr ur reply and fix :)

**Comment by v1s1t0r1sh3r3** [comment]
Did it worked on previous airgeddon versions? or you never tried? can you check on older airgeddon version? I want to know if is something broken from one version to the new or if is an existing problem on all.

More questions... rmnet_data0 is the phone integrated wireless interface or is the phone data interface?

**Comment by lightakun** [comment]
@v1s1t0r1sh3r3
2 months ago i tried with an older version..but same issue 

I tried the return 0 in that function in that script but still same problem

About the mobiledata interface, idont know how to check ! :(
Tell me any code to check it

**Comment by lightakun** [comment]
@v1s1t0r1sh3r3
Yes its a mobile data interface!
My mobile's wireless interface is wlan0

**Comment by v1s1t0r1sh3r3** [comment]
Did you tried using wireless interface (wlan0) as internet interface provider? does it work for you?

**Comment by lightakun** [comment]
@v1s1t0r1sh3r3
No. Actually I don't use WiFi here( I don't have any :p) .

**Comment by v1s1t0r1sh3r3** [comment]
You should try using wlan0 to know if it could be related. Try on your own network on which you have internet access on wireless interface, setting up another AP with different ESSID (used as target).

I ask for patience to check this. I'm on vacation with limited access to internet. I can't get into this yet. But we can gather all the needed info.

**Comment by lightakun** [comment]
@v1s1t0r1sh3r3
Okay np :)
Enjoy ur vacations

**Comment by v1s1t0r1sh3r3** [comment]
Hi, I'm back.

I tested airgeddon only with arm versions of Kali based on Raspberry Pi. I didn't tested it on Kali Nethunter. I'm trying to reproduce it but for now I'm not able to do that. I installed a Kali Nethunter on my smartphone but it is not officially supported (Oneplus 3). To install it it updated my phone's kernel and after that It worked.

Kali Nethunter seems to work but I'm not able to get vnc server working. And as I guess you know, without an X window system, airgeddon is not able to be run, so for now I can't reproduce it. I'm "fighting" with the vnc on my Nethunter but I'm not sure if this is not working for me because Oneplus3 is not supported by Kali Nethunter or if there is any other reason.

Did you finally tried using wlan0 as internet interface?
